clients/admin/src/components/StudentAddModalForm.tsx

```diff
@@ -237,6 +237,8 @@ const AddModalForm: React.FC<Props> = ({
                       minLength={11}
                       maxLength={11}
                       placeholder={"11122233344"}
+                      pattern={"[1-9][0-9]{10}"}
+                      title="0'la başlayamaz ve 11 haneli olmalıdır."
```

Comment:
```suggestion
                      pattern={"[1-9][0-9]{9}[02468]{1}"}
                      title="0'la başlayamaz, tek sayı ile bitemez ve 11 haneli olmalıdır."
```
TC kimlik tek sayı ile bitemez, böyle daha uygun olacaktır.

Reply:
Geç kalmışız 😔